cpp/ps0001_soin.cpp

```diff
@@ -1,31 +1,27 @@
-﻿
 #include <iostream>
 
 using namespace std;
 
 int main()
 {
-    int A;
-    int B;
-    int C;
-
-
-        cout << "Please enter the first Number A ? " << endl;
-        cin >> A; 
-
-        cout << "Please enter the first Number B ? " << endl;
-        cin >> B;
-
-        cout << "Please enter the first Number C ? " << endl;
-        cin >> C;
-
-        cout << "-----------------------------------------------" << endl;
-
-        cout << A + B + C << endl;
+    // prompting user for 3 numbers to add 1/2
+    short num1;
+    short num2;
+    short num3;
+    
+    cout << "Please enter the first Number A ? " << endl;
+    cin >> num1; 
 
+    cout << "Please enter the first Number B ? " << endl;
+    cin >> num2;
 
+    cout << "Please enter the first Number C ? " << endl;
+    cin >> num3;
 
+    // outputting the result of the summation process 3/3
+    cout << "-----------------------------------------------" << endl;
+    cout << num1 + num2 + num3 << endl;

```

Comment:
- [x] did summation process for the three inputted numbers
- [x] used `std::cout` in the right way to display output from the summation process
- [x] output is as required
```
155
940
168
--------
1263
```
+1 bonus mar as your program is user-friendly ⭐